src/utils.js

```diff
@@ -264,27 +266,17 @@ exports.getPriceBucketString = function(cpm) {
 };
 
 exports.mapForEach = function(obj, func) {
```

Comment:
We can probably deprecate this function as I think it is basically doing the same as `utils.each()`